### ISSUE TITLE
resource schematics

### ISSUE THREAD

**Issue by guiseek** [nrwl/nx#4456]
resource schematics (from nestjs, not nrwl / nest).

## Current Behavior
when using the nestjs schematics, resource function, returns this error:

`Property 'flat' does not match the schema. 'false' should be a 'undefined'.`

However the flat field is changed and continues with the same problem, I was in doubt if it opened here or in the nestjs repository.

## Expected Behavior

In the last version (10) it was working.

### Failure Logs
```sh
> Executing task in folder mycompany: ng generate @nestjs/schematics:resource --name=team --path=src/app --sourceRoot=apps/server --no-interactive --dry-run <

Property 'flat' does not match the schema. 'false' should be a 'undefined'.
The terminal process "/bin/bash '-l', '-c', '/Users/guiseek/dev/workspaces/mycompany/node_modules/.bin/ng generate @nestjs/schematics:resource --name=team --path=src/app --sourceRoot=apps/server --no-interactive --dry-run', 'ng generate @nestjs/schematics:resource --name=team --path=src/app --sourceRoot=apps/server --no-interactive --dry-run'" terminated with exit code: 1.
```

### Environment
 Node : 14.9.0
  OS   : darwin x64
  npm  : 6.14.10
  
  nx : Not Found
  @nrwl/angular : 11.0.20
  @nrwl/cli : 11.0.20
  @nrwl/cypress : 11.0.20
  @nrwl/devkit : 11.0.20
  @nrwl/eslint-plugin-nx : Not Found
  @nrwl/express : Not Found
  @nrwl/jest : 11.0.20
  @nrwl/linter : 11.0.20
  @nrwl/nest : 11.0.20
  @nrwl/next : Not Found
  @nrwl/node : 11.0.20
  @nrwl/react : Not Found
  @nrwl/schematics : Not Found
  @nrwl/tao : 11.0.20
  @nrwl/web : Not Found
  @nrwl/workspace : 11.0.20
  typescript : 4.0.5

**Comment by vsavkin** [comment]
Thank you for submitting the issue. 

This is the fix: https://github.com/nrwl/nx/commit/8a88fc8c71ae6e59fd65fbc0e4d84477b3322b4e

It will be in a patch release later this week.

**Comment by guiseek** [comment]
I wanted to correct it, but the last few days have been very busy at work, thanks for your attention!

**Comment by guiseek** [comment]
> Thank you for submitting the issue.
> 
> This is the fix: [8a88fc8](https://github.com/nrwl/nx/commit/8a88fc8c71ae6e59fd65fbc0e4d84477b3322b4e)
> 
> It will be in a patch release later this week.

I wanted to correct it, but the last few days have been very busy at work, thanks for your attention!
